src/components/Login.vue

```diff
@@ -0,0 +1,57 @@
+<script setup lang="ts">
+import { ref } from "vue"
+import { useCounterStore } from "../stores/counter";
+
+const store = useCounterStore();
+
+const storedName = ref("");
+const storedPassword = ref("");
+
+const textName = ref("");
+const textPassword = ref("");
+
+function submit(){
+  storedName.value = textName.value;
+  storedPassword.value = textPassword.value;
+}
```

Comment:
You should connect your login / submit form to a function that authenticates. This is kinda a good start, but we seem to have stopped halfway. You have the project spec, so you know what you should need to send the backend, what format it'll need to be in, what route you'd send it to, as well as what the expected return result is.

You could even write a dummy function to return a fake token so you can add the token to our app storage.

Speaking of app storage, once you get the token, you'll probably be storing it in a pinia store (either the default store or an auth store). Since we'll also probably be adding an action to the pinia store to submit / login, we should also store the password/username as state variables there.